demo/Tests/002-Cars/003-Check-Car-test.csx

```diff
@@ -0,0 +1,65 @@
+// Test 1: Car becomes available after 3 attempts
+tp.Test("Car should be available", () =>
+{
+    var checkCount = tp.GetVariable<int>("CarAvailabilityCheckCount");
+    checkCount++;
+    tp.SetVariable("CarAvailabilityCheckCount", checkCount);
+
+    var response = tp.Response;
+    var statusCode = response.StatusCode();
+
+    if (checkCount >= 3)
+    {
+        tp.Logger.LogInformation($"Car is now available after {checkCount} attempts!");
+        Equal(200, statusCode);
+    }
+    else
+    {
+        tp.Logger.LogInformation($"Car not available yet (attempt {checkCount})");
+        throw new Exception($"Car not available yet (attempt {checkCount})");
```

Comment:
Miesto exception by som dal asi `Assert.Fail("...")`. 
`Fail($"Car not available yet (attempt {checkCount})")`